CMake/Dependencies/libkvsCommonLws-CMakeLists.txt

```diff
@@ -6,10 +6,12 @@ include(ExternalProject)
 
 ExternalProject_Add(libkvsCommonLws-download
     GIT_REPOSITORY    https://github.com/awslabs/amazon-kinesis-video-streams-producer-c.git
-    GIT_TAG           v1.5.2
+    GIT_TAG           link-mbedtls-properly
```

Comment:
Should merge this fix on Producer C SDK and change this to master before we merge this into master.

Reply:
Want to highlight that `link-mbedtls-properly` branch is temporary. We need to do a release and update tag here again

Reply:
Right